docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,575 @@
+
+## Installing @next version
+
+To get started, install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
+
+**Follow these steps:**
+
+1. Meet the [software requirements for Zowe CLI](https://docs.zowe.org/stable/user-guide/systemrequirements.html#zowe-cli-requirements).
+
+2. To install or update the core CLI, open a command-line window and issue the following command:
+
+   ```
+   npm install -g @zowe/cli@next
+   ```
+
+3. Meet the [software requirements for each plug-in](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html#software-requirements-for-zowe-cli-plug-ins).
+
+4. (Optional) Check [npmjs.com](https://www.npmjs.com/) for any Zowe plug-ins that have an `@next` version available. If an `@next` version is available, you can issue the following command (substituting the plugin name) to install it. If no `@next` version, try installing the `@latest` version of the plug-in:
+
+    ```
+    zowe plugins install @zowe/<plugin-name>@next
+    ```
+
+   Zowe CLI and optional plug-ins are installed!
+
+5. If you previously had an instance of Zowe CLI installed, your old configuration files are no longer used in this version. Delete the following files from your local `.zowe/` directory:
+   - `.zowe/settings/imperative.json`
+   - `.zowe/profiles`
+
+   **Important!** Prior to deleting the contents of the `/profiles` directory, take note of any mainframe service details that you need (host, port, etc...). You might want to save the entire `/profiles` directory to another location on your computer so that you can reference or restore the profiles later.
+
+6. If you previously had the Secure Credential Store plug-in installed, uninstall it now to avoid unexpected behavior. Issue the following command:
+
+    ```
+    zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli
+    ```
+
+You can now configure the CLI and issue commands.
+## Initializing global configuration
+
+To begin, define a connection to z/OSMF and initialize your configuration files. We recommend this method for getting started, but you can choose create the configuration files manually if desired.
+
+**Follow these steps:**
+
+1. Issue the following command:
+
+   ```
+   zowe config init --global
+   ```
+
+   The CLI provides a series of prompts.
+
+2. Respond to the prompts to enter a service name, username, and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
+
+   After you respond to the prompts, the following file is added to your local `.zowe` directory:
+
+   - `zowe.config.json` - A global configuration file. This is the primary location where your MF service connection details such as host, port, etc... are defined. We recommend that you begin by working with this file.
```

Comment:
Not just recommend. If this config is not completed with MF service connection details such as host, port, etc., then they cannot proceed to step 3.

Reply:
Perhaps just point them to the editing configuration section